examples/ContainerApp/ContainerApp.Tests.ps1

```diff
@@ -63,7 +63,55 @@ Describe 'Verify Container Application' {
   }
 }
 
+Describe 'Verify Container Application Managed Environment' {
+  BeforeAll {
+    $Script:managedEnvName = 'managedenvbenchpresstest'
+    $Script:noManagedEnvName = 'nomanagedenv'
+  }
+
+  It "Should contain a Container Application Managed Environment named $managedEnvName - Confirm-AzBPResource" {
+    $params = @{
+      ResourceType      = "ContainerAppManagedEnv"
+      ResourceName      = $managedEnvName
+      ResourceGroupName = $rgName
+    }
+
+    (Confirm-AzBPResource @params).Success | Should -Be $true
+  }
+
+  It "Should contain a Container Application Managed Environment named $managedEnvName -
+    Confirm-AzBPResource" {
+    $params = @{
+      ResourceType      = "ContainerAppManagedEnv"
+      ResourceName      = $managedEnvName
+      ResourceGroupName = $rgName
+      PropertyKey       = "Name"
+      PropertyValue     = $managedEnvName
+    }
+
+    (Confirm-AzBPResource @params).Success | Should -Be $true
+  }
+
+  It "Should contain a Container Application named $managedEnvName" {
+    (Confirm-AzBPContainerAppManagedEnv -ResourceGroupName $rgName -Name $managedEnvName).Success | Should -Be $true
+  }
+
+  It "Should contain a Container Application named $managedEnvName" {
```

Comment:
You mind deleting this beDeployed test since it won't be applicable/a dupe of the one above

Reply:
Done with the merge from main.